polaris-for-vscode/package.json

```diff
@@ -20,7 +20,7 @@
     "shopify"
   ],
   "engines": {
-    "vscode": "^1.64.0",
```

Comment:
Does this change mean our minimum supported version of VSCode is now `^1.84.0`? Is there a reason we can't support the previous version for those who haven't updated yet?

Reply:
Oh this can most likely be reverted. Good shout out.